package/src/PrismicPreview.tsx

```diff
@@ -1,5 +1,8 @@
 import React, { useEffect } from "react";
 import { PrismicToolbar } from "@prismicio/react";
+import { getCookie } from "../lib/getCookie";
+import { useRouter } from "next/router";
+import { extractPreviewRefRepositoryName } from "../lib/extractPreviewRefRepositoryName";
```

Comment:
Small style fix: mixing internal and external imports is a bit messy. I organize imports like the following:

- All external imports at the top
- A line break
- All internal imports

It would look like this instead:

```ts
import React, { useEffect } from "react";
import { PrismicToolbar } from "@prismicio/react";
import { useRouter } from "next/router";

import { getCookie } from "../lib/getCookie";
import { extractPreviewRefRepositoryName } from "../lib/extractPreviewRefRepositoryName";
```